programs/ziti-prox-c/CMakeLists.txt

```diff
@@ -10,19 +10,32 @@ endif()
 target_link_libraries(ziti-prox-c PUBLIC ziti subcommand)
 target_include_directories(ziti-prox-c PRIVATE ${ziti-sdk_SOURCE_DIR}/inc_internal)
 
+set(ZITI_EXECUTABLE_NAME "ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}")
 if (UNIX)
     add_custom_target(ziti-prox-c-pack
-            BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}
+            BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/${ZITI_EXECUTABLE_NAME}
             DEPENDS ziti-prox-c
-            COMMAND tar zcf ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx} ziti-prox-c${CMAKE_EXECUTABLE_SUFFIX}
+            COMMAND tar zcf ${ZITI_EXECUTABLE_NAME} ziti-prox-c${CMAKE_EXECUTABLE_SUFFIX}
             )
 else()
-    add_custom_target(ziti-prox-c-pack
-            BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}
-            DEPENDS ziti-prox-c
-            COMMAND zip ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx} ziti-prox-c${CMAKE_EXECUTABLE_SUFFIX}
-            )
+    if(EXISTS "${ZITI_EXECUTABLE_NAME}")
+        add_custom_target(ziti-prox-c-pack
+                BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/${ZITI_EXECUTABLE_NAME}
+                DEPENDS ziti-prox-c
+                COMMAND zip ${ZITI_EXECUTABLE_NAME} ziti-prox-c${CMAKE_EXECUTABLE_SUFFIX}
+                )
+    else()
+        if(EXISTS "${CMAKE_BUILD_TYPE}/${ZITI_EXECUTABLE_NAME}")
+        add_custom_target(ziti-prox-c-pack
+                BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/${ZITI_EXECUTABLE_NAME}
+                DEPENDS ziti-prox-c
+                COMMAND ${CMAKE_BUILD_TYPE}/zip ${ZITI_EXECUTABLE_NAME} ziti-prox-c${CMAKE_EXECUTABLE_SUFFIX}
```

Comment:
is this the right path for zip?

Reply:
i thought so - i replaced all appropriate instances of ```ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}``` with ${ZITI_EXECUTABLE_NAME} (i thought)?